lib/nacelle-pushowl-plugin.js

```diff
@@ -32,11 +32,42 @@ const cartToPushOwlObject = (cart) => {
   }
 }
 
-const handleProductView = (product) => {
+const handleProductView = (product, currentVariant) => {
   let productId = parseInt(decodeBase64Id(product.pimSyncSourceProductId), 10)
+
   window.pushowl.trigger('syncProductView', {
     productId
   })
+
+  if (!currentVariant.availableForSale) {
+    window.pushowl.trigger('showWidget', {
+      type: 'backInStock',
+      product: {
+        id: product.id,
+        title: product.title
+      },
+      currentVariant: {
+        id: currentVariant.id,
+        name: currentVariant.name,
+        price: currentVariant.price,
+        available: currentVariant.currentVariant
+      }
+    })
+  } else if (currentVariant.availableForSale) {
+    window.pushowl.trigger('showWidget', {
+      type: 'priceDrop',
+      product: {
+        id: product.id,
+        title: product.title
+      },
+      currentVariant: {
+        id: currentVariant.id,
```

Comment:
Just ensure that these IDs are actual numeric IDs and not some base64 encoded string